File: Who ate the cookie?.py
def cookie(x):
    if isinstance(x,bool):
        return "Who ate the last cookie? It was the dog!"
    if isinstance(x,(int,float)):
        return "Who ate the last cookie? It was Monica!"
    if isinstance(x,str):
        return "Who ate the last cookie? It was Zach!"
        

File: Who ate the cookie?/Who ate the cookie?.js
function cookie(x) {
    if (typeof x === 'number') {
        return "Who ate the last cookie? It was Monica!";
    }
    else if (typeof x === 'string') {
        return "Who ate the last cookie? It was Zach!";
    }
    else {
      return "Who ate the last cookie? It was the dog!";
    }
}

